README.md

```diff
@@ -39,5 +39,4 @@ Helps you train at the blast furnace more efficiently
 1. Withdraw an ore to pick a method. If you withdraw both a metal and gold ore, the hybrid method will be selected.
     1. You can reset the method at any time by __shift-right clicking__ the method overlay
 2. Deposit the ores.
-3. Withdraw/equip required items as prompted by the plugin.
-4. Clear coal bag/glove placeholders and fill your bank with bank fillers, so you can use the deposit inventory button.
```

Comment:
> Replaced Deposit Inventory message with relevant message and item highlight for depositing ores or potions.

Not sure about this one - I think it's up to preference and should probably be configurable

Reply:
Fair enough, I've always done it the way I've coded it in this PR, but maybe the deposit inventory way is actually pretty good too and I haven't given it enough of a chance 🥲 

I will say, the way I have made it in this PR requires much less preparation to be good than the method involving bank fillers. Would it be possible to have this new way be the default one?

Reply:
I've taken that there thumb as a yes and have disabled using the Deposit Inventory button by default.